app/views/fact_check_confirm_error.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(layout: main_layout, govukBackLink : GovukBackLink)
+
+@(processId: String)(implicit request: Request[_], messages: Messages)
+
+@layout(messages("factCheckError.title")) {
+
+  @govukBackLink(BackLink(href = routes.FactCheckController.approval(processId).toString,
+    content = Text(messages("backlink.label"))))
+
+    <h1 class="govuk-heading-xl">@messages("factCheckError.heading")</h1>
+    <p class="govuk-body">@messages("factCheckError.info")</p>
+    <a href="@{routes.FactCheckController.approval(processId).toString}" aria-describedby="close" class="govuk-button" data-module="govuk-button">@messages("factCheckError.back")</a>
```

Comment:
I can't see the element with the id "close" referenced by aria-describedby.

Reply:
done, included in next commit